README.md

```diff
@@ -19,50 +19,70 @@ The OSHDB offers a distributed data base that splits storage and computation loa
     System.out.println(numberOfUsersEditingHighways);
 ```
 
+The OSHDB splits data storage and computations. It is then possible to use the [MapReduce](https://en.wikipedia.org/wiki/MapReduce) programming model to analyse the data in parallel, and optionally also on distributed databases. A central idea behind this concept is to bring the code to the data.
+
 OpenStreetMap History Data
 --------------------------
 
-OpenStreetMap contains a large variety of geographic data, for example differing in scale (ranging from single points of interests to whole country borders), or feature type (from concrete things like buildings up to more abstract concepts such as turn restrictions) and offers a lot of metadata (about individual contributors, modifications, etc.) that can be analyzed in a multitude of ways. At the same time, possibilities of working with this data are limited because of the large amount of needed resources and the lack of an easy-to-use analysis software. The huge information treasure in OSM for researchers, journalists, community members and other interested people is thereby kept hidden. A central goal of the OSHDB is to make this treasure available for a larger public.
+OpenStreetMap contains a large variety of geographic data, differing widely in scale and feature type. OSM contains everything from single points of interests to whole country borders, from concrete things like buildings up to more abstract concepts such as turn restrictions. OSM also offers metadata about the history and the modifications that are made to the data, which can be analyzed in a multitude of ways.
```

Comment:
also add a Link to the OSM wiki here? Although people reading this might be aware of what OSM-History is